src/test/java/org/plumelib/util/MathPlumeTest.java

```diff
@@ -462,6 +462,7 @@ public void test_modulus() {
 
     TestNonModulus testNonModulus = new TestNonModulus();
 
+    testNonModulus.checkStrict(new int[] {1}, null);
```

Comment:
This is a good test to add, but note that it is testing *non*modulus, whereas the method you changed is modulus.  It would be good to have tests with 0 and 2 elements as well.

Reply:
Mistake on my part, it appears I can't read. Will update with tests for modulus.

Reply:
I think I understand why I wrote this test; `MathPlume.modulusStrictInt` is called by [`MathPlume.nonmodulusStrictInternal`](https://github.com/plume-lib/plume-util/blob/master/src/main/java/org/plumelib/util/MathPlume.java#L1512); did you want me to create a test that directly tests `MathPlume.modulusStrictInt` in isolation?

Reply:
No, that is not necessary.